terraform/README.md

```diff
@@ -41,14 +44,14 @@ This project stores the terraform state in a remote storage in an azure storage
 
 When starting a new infrastructure you need to initialize the terraform state storage:
 ```bash
-$ cd azure-storage/
+$ cd azure-storage
```

Comment:
Earlier change added "/" to the end of 'cd' command, this one drops it. Make those consistent.